e2e/tests/hmctsAdministersCaseAfterSubmission_test.js

```diff
@@ -13,32 +13,37 @@ const c2Payment = require('../fixtures/c2Payment.js');
 const expertReportLog = require('../fixtures/expertReportLog.js');
 const dateFormat = require('dateformat');
 const dateToString = require('../helpers/date_to_string_helper');
-const mandatoryWithMultipleChildren = require('../fixtures/mandatoryWithMultipleChildren.json');
+const mandatoryWithMultipleChildren = require('../fixtures/caseData/mandatoryWithMultipleChildren.json');
 const supportingEvidenceDocuments = require('../fixtures/supportingEvidenceDocuments.js');
 
 let caseId;
 let submittedAt;
+let resetUser = true;
 
 Feature('Case administration after submission');
 
 BeforeSuite(async (I) => {
   caseId = await I.submitNewCaseWithData(mandatoryWithMultipleChildren);
   submittedAt = new Date();
-
-  await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId);
 });
 
-Before(async I => await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId));
+Before(async I => {
```

Comment:
tbh, I dont like this one. Looks like performance improvement. E2e should be really ease to read, thats why all of this I.see something. Should feel like plain english. It can cause problem if dev want to skip some tests.

Reply:
Yeah this is the one that I was not sure about either, I wanted to reduce the number of times logging in so I first just changed it from `navigate...As` to `navigate...` but then I noticed that some of the tests use switch users to la users. I guess a better way would be for the `steps_file.js` to remember the current logged in user and then if it is the same user not log in again.

Reply:
yeah, lets keep such improvement for later maybe

Reply:
sure thing, will revert the change here and then open another PR later for that